File: SoloLearn Codes/Kaleidoscopes/kaleidoscopes.c
#include <stdio.h>
#include <math.h>

int main() {
    int k;
    scanf("%d", &k);
    int cost = k * 5;
    double tax = cost * 0.07;
    printf("%.2f", ceil(((k > 1) ? (cost + tax - ((cost + tax) * 0.1)) : (cost + tax)) * 100.0) / 100.0);
    return 0;
}

File: SoloLearn Codes/Kaleidoscopes/kaleidoscopes.swift
import Foundation

let k = Int(readLine()!)!
let cost = k * 5
let tax = Double(cost) * 0.07
let discount = (Double(cost) + tax) * 0.1
let price = Double(cost)  + tax - (k > 1 ? discount : 0)
print(ceil(price * 100.0) / 100.0)